ts/theme/components/Header.ts

```diff
@@ -49,7 +50,11 @@ export default (): Theme => {
         }
       }
     },
-
+    minHeight:
+      variables.appHeaderHeight +
+      (isIphoneX()
```

Comment:
why can't we use [this](https://www.npmjs.com/package/react-native-iphone-x-helper#getstatusbarheightsafe) with the `safe` param?

Reply:
changed with suggested method